HelpMyStreetFE/HelpMyStreetFE/Controllers/YotiController.cs

```diff
@@ -26,17 +26,18 @@ public YotiController(IOptions<YotiOptions> options, IValidationService validati
 
         public IActionResult Authenticate()
         {
-            var viewModel = new AuthenticateViewModel { ClientSdkId = _options.ClientSdkId, DomId = _options.DomId, ScenarioId = _options.ScenarioId };
+            var viewModel = new AuthenticateViewModel { UserId = HttpContext.User.FindFirst(ClaimTypes.NameIdentifier).Value, ClientSdkId = _options.ClientSdkId, DomId = _options.DomId, ScenarioId = _options.ScenarioId };
             return View(viewModel);
         }
 
-        [HttpPost]
-        public async Task<IActionResult> ValidateToken([FromBody] ValidateTokenRequest request, CancellationToken token = default)
+        [HttpGet]
+        public async Task<IActionResult> ValidateToken(string userId, string token, CancellationToken cancellationToken)
         {
-            var id = HttpContext.User.FindFirst(ClaimTypes.NameIdentifier).Value;
-            if (id != null)
+            var id = userId ?? HttpContext.User.FindFirst(ClaimTypes.NameIdentifier).Value;
+
+            if (id != null && token!= null)
```

Comment:
If you look in the validation service it already does the token check and a few other checks